src/Symfony/Bridge/PhpUnit/Legacy/ForwardCompatTestTraitForV5.php

```diff
@@ -79,4 +79,59 @@ private function doTearDown()
     {
         parent::tearDown();
     }
+
+    public static function assertIsArray($actual, string $message = ''): void
```

Comment:
the polyfill must be compatible with PHP 5.5 :)

Reply:
fixed. fabbot is not happy because of `@return void` in previous methods. SHould I remove theme?

Reply:
we should keep them as they are useful to DebugClassLoader (allows triggering a deprecation to hint ppl they should add `void`)